ios/RNSentry.mm

```diff
@@ -7,19 +7,22 @@
 #import "RCTConvert.h"
 #endif
 
+#if __has_include(<hermes/hermes.h>)
+#define SENTRY_PROFILING_ENABLED 1
+#define SENTRY_TARGET_PROFILING_SUPPORTED 1
+#else
+#define SENTRY_PROFILING_ENABLED 0
+#define SENTRY_TARGET_PROFILING_SUPPORTED 0
+#endif
```

Comment:
Does `PrivatesHeader.h` get included into the native portion of the RN SDK? If so, that should pull in `SentryProfilingConditionals.h` which defines `SENTRY_TARGET_PROFILING_SUPPORTED`. Or does `SentryProfilingConditionals.h` get included in any other way?

Reply:
Shouldn't we just use a single definition here? I only saw  `SENTRY_PROFILING_ENABLED` being used in the code. 
Is `SENTRY_TARGET_PROFILING_SUPPORTED` being used elsewhere?

Reply:
`SENTRY_TARGET_PROFILING_SUPPORTED` is used in the native profiler in `sentry-cocoa`.

RN SDK has `sentry-cocoa` as a dependency so we don't have access to private headers.

Reply:
We can include it directly https://github.com/getsentry/sentry-react-native/pull/3349/commits/4e85b3dccb240b09132bc529ae90c00e428b3cdc